pkg/trigger/discovery_handler_test.go

```diff
@@ -18,17 +18,16 @@ package trigger
 
 import (
 	"fmt"
-	"reflect"
-	"strings"
-	"testing"
-
 	"github.com/kubernetes-sigs/kube-storage-version-migrator/pkg/apis/migration/v1alpha1"
 	"github.com/kubernetes-sigs/kube-storage-version-migrator/pkg/clients/clientset/fake"
```

Comment:
I'd maybe cut these two out to a separate block, too

Reply:
Done